### PR TITLE
Remove placeholder image from AMP ads

### LINKED ISSUE #6871: AMP ads - remove placeholder image
**Is your feature request related to a problem? Please describe.**
During development, a placeholder image has been shown while the ad is loading. This should be replaced with an empty space.

**Describe the solution you'd like**
- Remove the placeholder image from https://github.com/bbc/simorgh/blob/latest/src/app/containers/Ad/Amp/index.jsx#L139 and https://github.com/bbc/simorgh/blob/latest/src/app/containers/Ad/Amp/index.jsx#L148
- Ensure that the space for the ad is still reserved when the image has been removed, due us passing in the width and height to the amp-ad component
- Remove all references to 'https://via.placeholder.com' in the content security policy.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Snapshot tests are updated, placeholder image is no longer rendered.

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #6871

**Overall change:**
Removed placeholder image from AMP ads and all existing instances of `via.placeholder.com`.

**Code changes:**
- Removed `<amp-img/>` from `<amp-ad/>`
- Added height and width values to `<amp-ad/>`
- Removed `https://via.placeholder.com` from `cspHeader/index.js` and all related tests.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Ad/Amp/index.jsx
Comment (tochwill) [inline src/app/containers/Ad/Amp/index.jsx]: I think the width and height should already be coming from the `ampAdPropsMobile` and `ampAdPropsMobile` that are being spread here!

**Review by tochwill — COMMENTED** [review]
Looks good! Just one comment :)

**Comment by hotinglok** [comment]
For anyone testing this, the ad seems to show up sometimes and sometimes not. This is partially fixed by removing ` <div amp-access="toggles.ads.enabled" amp-access-hide="true">`. It has to do with the toggles service which is being updated in #6915

**Review by tochwill — APPROVED** [review]

**Review by mmcculloch — APPROVED** [review]

**Comment by mmcculloch** [comment]
Nice! I'll wait for this to be merged before merging https://github.com/bbc/simorgh/pull/6931
There will be a few conflicts to resolve

**Review by DenisHdz — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me empty space is seen before the ad loads.

**Pull request merged**
